homeassistant/strings.json

```diff
@@ -52,6 +52,7 @@
         "invalid_access_token": "Invalid access token",
         "invalid_api_key": "Invalid API key",
         "invalid_auth": "Invalid authentication",
+        "invalid_credentials": "Invalid credentials",
```

Comment:
Usually we use invalid_auth for this case.

Reply:
ok, I'll replace it